release.config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   branch: 'master',
-  repositoryUrl: 'https://github.com/saltstack-formulas/rabbitmq-formula',
+  repositoryUrl: 'https://github.com/saltstack-formulas/template-formula',
```

Comment:
I'm not sure but I think `release.config.js` must point to `rabbitmq-formula`

Reply:
yes. fixed that too. thanks.